packages/strapi-versioning-plugin/src/services/Versioning.ts

```diff
@@ -0,0 +1,50 @@
+import { Context } from "koa";
+
+type Entity = any;
+type Data = Array<Entity>;
+
+const isModelExists = (ctx: Context): boolean =>
+  (ctx.params !== undefined && ctx.params.model !== undefined) ||
+  Object.values(global.strapi.models).find(
+    (el) => el.collectionName == ctx.url.split("/")[1],
+  ) !== undefined;
+
+const findUid = (ctx: Context): string =>
+  ctx.params.model
+    ? global.strapi.db.getModel(ctx.params.model).uid
```

Comment:
I think db.getModel could return null or udnefined -> so this could throw an error, please change it and improve typings

Reply:
It cant return null or undefined because if params.model.exists (strapi app creating ctx with params.model from its page) it has to be in db.